src/clr-addons/themes/phs/theme.clarity.scss

```diff
@@ -2,7 +2,7 @@
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
 
-[cds-theme~='phs'] {
+:root [cds-theme~='phs'] {
```

Comment:
I think this was just removed by @derkoe in the v16 commit, why are we adding it back again?

Reply:
@thred yes please do not change this back otherwise the themes won't work when set on the body tag

Reply:
It simply does not work, it he :root is left away. Please call me, in order to optimize this setting ...

Reply:
just tried - seems this is handled differently with Angular/Clarity 17 but I do not know why. 

Angular/Clarity 17 this works
```css
:root[cds-theme~='phs']
```

with Angular 16/Clarity 16 only this works
```css
[cds-theme~='phs']
```
Any idea why?

Reply:
Nope, just tried it again. Removed the :root form the other two locations, it truly wasn't necessary there.

Reply:
I think, we can resolve this ...